integ-test/src/test/java/org/opensearch/sql/sql/DateTimeFunctionIT.java

```diff
@@ -513,6 +513,16 @@ public void testHourFunctionAliasesReturnTheSameResults() throws IOException {
     result1.getJSONArray("datarows").similar(result2.getJSONArray("datarows"));
   }
 
+  @Test
+  public void testLastDay() throws IOException {
+    JSONObject result = executeQuery(
+        String.format("SELECT last_day(date0) FROM %s LIMIT 3", TEST_INDEX_CALCS));
```

Comment:
Could add `time0` and `datetime0` to test time and timestamp.

Reply:
+1
with cast to ensure that different types are supplied to the function

Reply:
Added here using casting to ensure that the types passed in are correct https://github.com/Bit-Quill/opensearch-project-sql/pull/218/commits/22be4c7d0ff0a56469bb6f90383b7a35daf23e16